robot/basestation/app.py

```diff
@@ -186,8 +187,9 @@ def initiate_feed_recording():
 @app.route("/stop_feed_recording/", methods=["POST", "GET"])
 def stop_feed_recording():
     stream_url = request.args['stream_url']
+    rotation = request.args['camera_rotation']
     if is_recording_stream(stream_url):
-        success, message = stop_recording_feed(stream_url)
+        success, message = stop_recording_feed(stream_url, rotation)
```

Comment:
from a software engineering perspective, I think that it is not the purpose of "stop_recording_feed" to be fed a rotation. I would remove the argument and pass it in the hashmap